src/components/StepsList/index.js

```diff
@@ -74,7 +74,7 @@ const StepsList = ({
       return 'Please enter the necessary info below.';
     }
     if (activeStep === 1 && !availableSteps[1]) {
-      return 'Please select at least 2 plants.';
+      return 'Please select at least 1 plants.';
```

Comment:
should be `Please select at least 1 plant`

Reply:
I have updated this, I also noticed that the caption text on the page should change, what about `Select your species.`?
![image](https://github.com/precision-sustainable-ag/dst-seedcalc/assets/29057898/7f39424c-d7ae-43c5-983c-b24073718b6e)

Reply:
Select one or more species for your mix.